docker/test/stateless/run.sh

```diff
@@ -231,10 +232,16 @@ do
     fi
 done
 
+data_path_config="--path=/var/lib/clickhouse/"
+if [[ -n "$USE_S3_STORAGE_FOR_MERGE_TREE" ]] && [[ "$USE_S3_STORAGE_FOR_MERGE_TREE" -eq 1 ]]; then
+    # We need s3 storage configuration (but it's more likely that clickhouse-local will fail for some reason)
```

Comment:
Out of curiosity, which are they?

Reply:
@Felixoid interesting, after my comment in this PR I got this from CI / `MergeOnApproval`:

```
2023-11-22 20:03:14,680 User robot-ch-test-poll1 with 4807 remaining requests is used
Traceback (most recent call last):
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/merge_pr.py", line 294, in <module>
    main()
  File "/home/ubuntu/actions-runner/_work/ClickHouse/ClickHouse/tests/ci/merge_pr.py", line 233, in main
    repo.organization._url.value = repo.organization.url.replace(  # type: ignore
AttributeError: can't set attribute 'value'
Error: Process completed with exit code 1.
```

Reply:
it's fixed in master for a week